### PR TITLE
Add CVE mentions with brief descriptions to Docker Deskop release notes

### PR DESCRIPTION
**Pull request by mat007**
### Proposed changes

Add CVE mentions with brief descriptions to Docker Deskop release notes

### CODE REVIEW

File: desktop/mac/release-notes/index.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### Security
+
+- Fixed [CVE-2021-44719](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-44719) where Docker Desktop could be used to access any user file on the host from a container, bypassing the allowed list of shared folders.
```
Comment (usha-mandya) [inline desktop/mac/release-notes/index.md:38]:
```suggestion
- Fixed [CVE-2021-44719](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-44719){: target="_blank" rel="noopener" class="_"} where Docker Desktop could be used to access any user file on the host from a container, bypassing the allowed list of shared folders.
```

File: desktop/windows/release-notes/index.md [hunk 2]
```diff
@@ -32,6 +32,11 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){:
 > .button .primary-btn }
 
+### Security
+
+- Fixed [CVE-2022-23774](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-23774) where Docker Desktop allows attackers to move arbitrary files.
```
Comment (usha-mandya) [inline desktop/windows/release-notes/index.md:37]:
```suggestion
- Fixed [CVE-2022-23774](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-23774){: target="_blank" rel="noopener" class="_"} where Docker Desktop allows attackers to move arbitrary files.
```

File: desktop/windows/release-notes/index.md [hunk 3]
```diff
@@ -32,6 +32,11 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Fixed [CVE-2021-44719](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-44719) where Docker Desktop using the Hyper-V backend could be used to access any user file on the host from a container, bypassing the allowed list of shared folders.
```
Comment (usha-mandya) [inline desktop/windows/release-notes/index.md:38]:
```suggestion
- Fixed [CVE-2021-44719](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-44719){: target="_blank" rel="noopener" class="_"} where Docker Desktop using the Hyper-V backend could be used to access any user file on the host from a container, bypassing the allowed list of shared folders.
```

Comment (stephen-turner) [inline desktop/windows/release-notes/index.md:38]: Are we sure this issue occurred on Windows too? I know the grpcfuse change also applies to Hyper-V, but I thought that the security issue was only on Mac. I'm not 100% sure though.
(Ref: https://docs.google.com/document/d/1ZqXppY5pIdms21ird3O7VvqHrqY6efIFiPm6SuMfOYg/edit?usp=sharing)

Comment (mat007) [reply desktop/windows/release-notes/index.md:38]: I wondered as well. The docs says Mac Intel only, so maybe let’s stick with that.
@djs55 would know 😁

**Pull request merged**